pkg/rendering/renderer.go

```diff
@@ -85,12 +85,17 @@ func RenderTemplates(backplaneConfig *v1alpha1.BackplaneConfig, images map[strin
 	log := log.FromContext(context.Background())
 	var templates []*unstructured.Unstructured
 	errs := []error{}
-
+	backplaneOperatorNamespace := ""
 	chartDir := chartsDir
 	if val, ok := os.LookupEnv("DIRECTORY_OVERRIDE"); ok {
 		chartDir = path.Join(val, chartDir)
 	}
-
+	if val, ok := os.LookupEnv("POD_NAMESPACE"); ok {
+		backplaneOperatorNamespace = val
+	} else {
+		log.Info(fmt.Sprintf("error retrieving namespace"))
+		return nil, errs
```

Comment:
No error is being returned here, despite failing to retrieve a namespace being a critical failure point